docs/sql-ref-syntax-aux-cache-cache-table.md

```diff
@@ -19,4 +19,65 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+`CACHE TABLE` statement caches the table's contents with the given storage level. This reduces scanning of the original files in future queries.
+
+### Syntax
+{% highlight sql %}
+CACHE [ LAZY ] TABLE table_name
+    [ OPTIONS ( 'storageLevel' [ = ] value ) ] [ [ AS ] query ]
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>LAZY</em></code></dt>
+  <dd>Only cache the table when it is first used, instead of immediately.</dd>
+</dl>
+
+<dl>
+  <dt><code><em>table_name</em></code></dt>
+  <dd>The name of the table to be cached.</dd>
+</dl>
+
+<dl>
+  <dt><code><em>OPTIONS ( 'storageLevel' [ = ] value )</em></code></dt>
+  <dd>
+  <code>OPTIONS</code> clause with <code>storageLevel</code> key and value pair. If a key other than <code>storageLevel</code> is used, a WARN with message "org.apache.spark.sql.execution.command.CacheTableCommand: Invalid options: XXXX -> XXXX" will be issued. The valid options for <code>storageLevel</code> are:
```

Comment:
@huaxingao Do we need to detail the exact warning message ? We could say "An warning is issued when .... event occurs" ?